addons/mail/static/src/xml/abstract_thread_window.xml

```diff
@@ -5,7 +5,7 @@
         @param {mail.AbstractThreadWindow} widget
     -->
     <t t-name="mail.AbstractThreadWindow">
-        <div class="o_thread_window o_in_home_menu" t-att-data-thread-id="widget.getID()">
+        <div t-attf-class="o_thread_window o_in_home_menu o_thread_window_#{widget._thread and widget._thread._type}" t-att-data-thread-id="widget.getID()">
```

Comment:
Avoid automatically crafted classes because those are very hard to maintain (impossible to find it back in the code from the full name). It's preferable to add conditions and explicit names.

Using private variables in a template is also not a good sign.

Reply:
@seb-odoo I added the type in attribute, so I can select it in CSS without crafting classes ;)